### ISSUE TITLE
Prod French Dashboard Feedback

### ISSUE THREAD

**Issue by AnnieMungai** [OpenSRP/opensrp-client-unicef-tunisia#163]
- [ ] The months seem not to load all of the months of the calendar.  
![image](https://user-images.githubusercontent.com/58034047/147183552-560411ca-e6a6-445f-8132-a81e5311781a.png)

**Comment by muneneg** [comment]
@AnnieMungai Yes, this loads the data that is in the tables. Only months that have data will load.
